src/System.IO/src/netstandard15aot/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "frameworks": {
+    "netstandard1.5": {
```

Comment:
Agreed. The project.json should just be netstandard1.3 for the framework. netstandard13aot is just shorthand for our build system.